### ISSUE TITLE
Failure to read this file

### ISSUE THREAD

**Issue by keflavich** [astrofrog/pyavm#32]
This file has an xmp packet:

![outflow_co_redblue_naco_green_asinhgreen](https://cloud.githubusercontent.com/assets/143715/12265424/3a9aa1ba-b93d-11e5-9b63-aff7f1a54fc1.png)

The first few lines of the file:

```
<89>PNG^M
^Z
^@^@^@^MIHDR^@^@^Q¨^@^@^Q¤^H^B^@^@^@":~é^@^@^P[iTXtXML:com.adobe.xmp^@^@^@^@^@<?xpacket begin="ï»¿" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:avm="http://www.communicatingastronomy.org/avm/1.0/"
```

I'm reading this in py3.  I am not sure what's going on, but there is clearly some issue with the reader.

**Comment by keflavich** [comment]
It's a py3 problem; I have the solution in https://github.com/astrofrog/pyavm/pull/31, which is just to compare the string to a bytestring

**Comment by astrofrog** [comment]
Should be fixed in PyAVM v0.9.3
